scss/details.scss

```diff
@@ -4,11 +4,38 @@
   font-family: 'Roboto';
 }
 
+.back-home {
+  align-self: flex-start;
+  font-size: 18px;
+  margin: 0px 0px 50px 150px;
+  position: relative;
+
+  &::before {
+    background: url('assets/back-arrow.png') no-repeat;
```

Comment:
In this case it works because the image is small, but you should have present that the correct way is using `background-image` property